guides/doc-Provisioning_Guide/topics/app-Deploy_FIPS_hosts.adoc

```diff
@@ -83,10 +76,12 @@ The FIPS provisioning templates are provided in a git repository. In this proced
 
 . On the {ProjectServer}, clone the git repository containing the FIPS enabled templates, then change into the repository's directory.
 +
+ifeval::["{Build}" == "satellite"]
```

Comment:
Why are these satellite-only? I'd expect them to work on Foreman as well. My only question is why these are not part of community-templates so we ship them to every user.

The whole section refers to this repository and doesn't really make sense without it.

@lzap can we just integrate these templates upstream which means we'll eventually ship them out of the box?

Reply:
Because it is the Satellite github @ekohl - I need the link for the Foreman community templates, hence the WIP :D

Reply:
PS: they did not make it into the regular shipped templates in Satellite, so we were asked to link to the Github page. I am relying on you a bit to let me know whether they are in Foreman, if so, I can write a Foreman-specific section.

Reply:
While it may be the Satellite org, this is publicly available so I'd release this without hiding it. The proper upstreaming would probably mean they become part of all the existing files and there would be no need for anything other than setting the `fips_enabled` parameter to `true` on the host to provision. That should then flow back into downstream and the Satellite customer would only need to set that one parameter. At least, that's how I'd see this feature if it was properly finished.

Reply:
By that logic, I think most of the changes in this PR can be reverted, because all the templates will have Satellite rather than Foreman if we are just pulling straight from the Satellite github and if none of the templates are available in the upstream files. 

By the sounds of it, what is required is an ifeval with a note for Foreman saying that they can use the Satellite resources for this. 

What do you think @ekohl @lzap ?

Reply:
So it looks like what I suggested has already happened and this entire chapter is downstream-only. When those changes make it into downstream, the entire chapter can also be dropped.

Reply:
Well I think this is just a proposal and we can come to agreement. It does not make any sense to hide it.